src/parser/jobs/war/StormsEye.js

```diff
@@ -0,0 +1,88 @@
+import React, {Fragment} from 'react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+import {Rule, Requirement} from '../../core/modules/Checklist'
+import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+const STORMS_EYE_DURATION = 30000
+const STORMS_EYE_BUFFER = 10000
+
+export default class StormsEye extends Module {
+	static handle = 'stormseye'
+	static dependencies = [
+		'checklist',
+		'combatants',
+		'invuln',
+		'cooldowns',
```

Comment:
Is the cooldowns dependency even used? 👀

Reply:
Was in an earlier version, not anymore. xd